sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.2 (Unreleased)
 
+- Bug fix: Replaced cases where `instanceof` was used to compare errors with comparisons on the error names. Uneven updates on our dependencies would cause `instanceof` to mismatch our error comparisons. Comparing by error name makes sure dependency changes won't alter our logic.
```

Comment:
Can we specify when this issue got introduced? For example if this is something that was introduced in v2 beta 1, then v1 users in the future will not have to care about it

Reply:
v1 had these instances of `instanceof`:

```
src/credentials/managedIdentityCredential/index.ts:179:      if (err instanceof CredentialUnavailable) {
src/credentials/managedIdentityCredential/imdsMsi.ts:86:          (err instanceof RestError && err.code === RestError.REQUEST_SEND_ERROR) ||
src/credentials/interactiveBrowserCredential.ts:100:      if (e instanceof AuthenticationRequired) {
src/credentials/interactiveBrowserCredential.browser.ts:97:      if (err instanceof msal.AuthError) {
src/credentials/deviceCodeCredential.ts:131:      if (e instanceof AuthenticationRequired) {
src/credentials/chainedTokenCredential.ts:67:        if (err instanceof CredentialUnavailable) {
```

Instanceof was an issue only on `interactiveBrowserCredential.browser.ts`, which uses the older MSAL that is deprecated. For v1, I don't think this is an issue we should worry about at this point.

Should I remove the changelog entry then?

Reply:
In that case, we can skip the changelog entry. Not much use to the end user